### ISSUE TITLE
Might be causing build issue on Flutter master channel 3.9.0-15.0.pre.22

### ISSUE THREAD

**Issue by enqvida** [idootop/custom_nested_scroll_view#9]
I'm getting ane error (Xcode):
../../../.pub-cache/git/custom_nested_scroll_view-74a3a7163278c8e84
18ba359d94f6448196d0b25/lib/src/nested_scroll_view.dart:526:24:
Error: The method '_NestedScrollMetrics.copyWith' doesn't have the
named parameter 'devicePixelRatio' of overridden method 'Object
with ScrollMetrics.copyWith'.

I have no issues on stable.

**Comment by idootop** [comment]
Thank you for your feedback. I noticed this issue a few months ago, but the related changes were only in the flutter master branch at that time and had not yet stabilized. It looks like this change has made its way into the flutter stable branch, so it's time to start addressing it. I will take a look this weekend. :)

**Comment by idootop** [comment]
Fixed, now you can use the `flutter-3.7` branch. :)

```shell
dependencies:
  ...
  custom_nested_scroll_view:
    git:
      url: https://github.com/idootop/custom_nested_scroll_view.git
      # Which branch to use is based on your local flutter version
      ref: flutter-3.7 # flutter-2.x flutter-3.0 flutter-3.4 flutter-3.7
```

| Git branch      | Supported flutter versions       |
| --------------- | -------------------------------- |
| flutter-3.7     | >=3.7.0-13.0.pre                 |
| flutter-3.4     | >=3.4.0-27.0.pre <3.7.0-13.0.pre |
| flutter-3.4-pre | >=3.4.0-17.0.pre <3.4.0-27.0.pre |
| flutter-3.0     | >=2.12.0-4.0.pre <3.4.0-17.0.pre |
| flutter-2.x     | <2.12.0-4.0.pre                  |
